src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -75,6 +83,14 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
 
   const handleClose = () => {
     setModalType(null)
+    trackEvent({
+      action_type: "clickedCloseValidationAddress",
+      context_module: ContextModule.ordersShipping,
+      context_page_owner_type: OwnerType.ordersShipping,
+      context_page_owner_id: contextPageOwnerSlug,
+      option: "null",
+      label: "X",
```

Comment:
in this case, if I consider both pressing "X" and clicking away from the modal the same action, is it okay to specify null as the `option` and `label`? The other reason why I am conflating the two is that its becoming difficult to figure out where to make the distinction in` handleClose()`. I honestly can't tell when the modal x button is being pressed or clicked away from in the code.

Reply:
Agreed! So if option and label are `null` , I can assume the user didn't click on the buttons but closed the modal?

Reply:
yes, I made an assumption and chose `null` as the qualifier here, hope that's okay!

Reply:
Adding [here](https://docs.google.com/spreadsheets/d/1c65vugkF2C4YuXwBXno8aqsBh4B2fTwxiLmgY_MhjWc/edit#gid=0) to make sure we are documenting it.

Reply:
Sorry, actually I chose "X" but let me know if that is inconsistent

Reply:
I can switch both option and label to `null` instead of `X` to reflect the documentation. Plus, `null` makes it easier to determine that is represents both an X and click away.

Reply:
My most recent commit reverted option and label for modal closing tracking actions to `null` so going to resolve this conversation if no objections.